pkg/sql/colexec/colexecutils/BUILD.bazel

```diff
@@ -73,3 +75,8 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+disallowed_imports_test(
+    "colexecutils",
+    disallowed_prefixes = ["//pkg/sql/colexec"],
```

Comment:
:( so I did something weird here, and maybe I regret it. This parameter takes a go import path and not a bazel target. The reason I did this is that we have some paths which aren't go libraries and thus don't have valid prefixes. Here's what I think I'll do: make it an error to not either have the prefix be `pkg` or `github.com/cockroachdb/cockroach/pkg` and then if it's the former, convert it to the latter.

Reply:
by "have the prefix be" I mean the prefix of the prefix